src/ImageSearchModal.jsx

```diff
@@ -38,11 +40,15 @@ export class ImageSearchModal extends React.Component {
             this.activeConnection.close();
     }
 
+    onToggleUser() {
+        this.setState({ isRoot: !this.state.isRoot });
```

Comment:
While this seems to work, it feels kind of brittle -- wouldn't it be better to get the `event` and check its target.checked (or so) as to whether it's actually on?